README.md

```diff
@@ -215,9 +186,9 @@ All projects should be integrated with a continuous integration server. Unless y
 
 ### Project Participants
 
-Designing and working alone on significant software project is not good. If this is the case go to management and kick them in the nuts.
+Designing and working alone on a significant software project is not good. If this is the case go to management and kick them in the nuts.
 
-If circumstances finds you working alone, designate a "co-pilot" who has basic familiarity with the project and can provide input and code reviews.
+You will often be working alone on a signifcant software project. To avoid being disturbed when you're sipping coctkails in Syden, make your code readable, provide good documentation, and be generous when _you_ need to dig into another person's solo project because they're off sipping cocktails in Syden.
```

Comment:
```suggestion
You will often be working alone on a significant software project. To avoid being disturbed when you're sipping cocktails in Syden, make your code readable, provide good documentation, and be generous when _you_ need to dig into another person's solo project because they're off sipping cocktails in Syden.
```